apps/g2-oidc/03-openstack/03-openstack.base/05-rabbitmq.yaml

```diff
@@ -0,0 +1,57 @@
+
+
```

Comment:
2 useless empty lines.

Reply:
All unneeded empty lines in k8s/CRD manifests except empty lines in raw encoded config files were dropped by 04a287d016f855e488ed0d5ac6fc917c145f7c0c.